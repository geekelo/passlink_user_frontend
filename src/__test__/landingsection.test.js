import React from 'react';
import { render } from '@testing-library/react';
import LandingSection from '../homePage/javascripts/landingsection';

test('renders LandingSection correctly', () => {
  const { asFragment } = render(<LandingSection />);

  // Initial render snapshot
  expect(asFragment()).toMatchSnapshot();
});
