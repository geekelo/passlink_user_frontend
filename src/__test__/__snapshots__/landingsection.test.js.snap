// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LandingSection correctly 1`] = `
<DocumentFragment>
  <div
    class="body-section"
  >
    <div
      class="arrow-section"
    >
      <button
        aria-label="Previous Button"
        class="arrow"
        type="button"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
          />
        </svg>
      </button>
      <button
        aria-label="Next Button"
        class="arrow active-arrow"
        type="button"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="m12 4-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
          />
        </svg>
      </button>
    </div>
    <div
      class="body-slide"
    >
      <div
        class="slide-content"
      >
        <h1
          class="title"
        >
          Find online & onsite events on PassLink
        </h1>
        <div
          class="ticks-cta-section"
        >
          <div
            class="ticks-section"
          >
            <p>
              <span
                class="ticks"
              >
                ✔
              </span>
              <span>
                Enhance your knowledge, skills, and career prospects
              </span>
            </p>
            <p>
              <span
                class="ticks"
              >
                ✔
              </span>
              <span>
                Participate in celebrations & social gatherings
              </span>
            </p>
            <p>
              <span
                class="ticks"
              >
                ✔
              </span>
              <span>
                Engage extensive collection of religious events
              </span>
            </p>
            <p>
              <span
                class="ticks"
              >
                ✔
              </span>
              <span>
                Access mentorship sessions & programs
              </span>
            </p>
          </div>
          <button
            class="btn"
            type="submit"
          >
            <a
              href="v.com"
            >
              <span>
                FIND EVENTS  
              </span>
              <i
                class="material-icons"
              >
                travel_explore
              </i>
            </a>
          </button>
        </div>
      </div>
      <div
        class="infographic-section"
      >
        <img
          alt="infographic"
          class="infographic"
          src="infographic1.png"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
